<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiversityMeasureDlg</class>
 <widget class="QDockWidget" name="DiversityMeasureDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>117</height>
   </rect>
  </property>
  <property name="allowedAreas">
   <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
  </property>
  <property name="windowTitle">
   <string>Diversity measure</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lblDistTypeOfMeasure">
        <property name="text">
         <string>Type of measure:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cboMeasureType">
        <item>
         <property name="text">
          <string>Unweighted</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weighted, unnormalized</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Weighted, normalized</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblDistSplits">
        <property name="text">
         <string>Measure:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cboMeasure"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="btnRun">
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
